### DISCUSSION TITLE
How to handle html tags like <p>, <em>, <br> etc. ?

### DISCUSSION THREAD

**Discussion by adi286sadsd** [pocka/slack-message-parser#37] [Q&A]
I have a string like `<p>this is a <em>meeting<em><p>`. How do i parse this into nodes? 
Currently i get this error in combinator.ts 
`text.charAt is not a function
    at Array.<anonymous> (combinator.ts:46:25)
    at combinator.ts:9:30
    at parseText (mod.ts:28:19)
    at parse (mod.ts:57:15)`

**Comment by pocka** [comment]
This is a parser for [Slack's `mrkdwn` message format](https://api.slack.com/reference/surfaces/formatting).
To parse HTML strings, use HTML parser such as [`DOMParser (Web API)`](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser) and [`node-html-parser`](https://www.npmjs.com/package/node-html-parser).
